scripts/barclamp_mgmt_lib.rb

```diff
@@ -468,11 +435,11 @@ def bc_install_layout_1_app(from_rpm, bc, bc_path, yaml)
   end
 
   # copy over the crowbar YAML file, needed to update catalog
-  yml_barclamp = get_crowbar_yml_path(bc_path)
+  yml_barclamp = get_yml_paths(bc_path, bc)
   yml_path = File.join CROWBAR_PATH, 'barclamps'
   yml_created = File.join(yml_path, "#{bc}.yml")
   FileUtils.mkdir yml_path unless File.directory? yml_path
-  FileUtils.cp yml_barclamp, yml_created
+  FileUtils.cp yml_barclamp, yml_path
```

Comment:
Oh, is it because it's copying multiple files? In which case `yml_barclamp` should probably be `yml_barclamps`.